### PR TITLE
get ready for code review

### PR DESCRIPTION
**Pull request by billymohajeri**

### CODE REVIEW

**Comment by billymohajeri** [comment]
## STATUS: REQUIRES CHANGES 🚦 ♻️ 
Hi @billymohajeri, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Informative commit messages
- GitHub flow used
- Good PR

## Required Changes ♻️
- You have some errors in liter. Please solve those.
- It's better to have a descriptive README.md file.
- It's better to use semantic HTML tags and put every different block of codes in specified sections.

**Review by anagudelogu — CHANGES_REQUESTED** [review]
Hi @billymohajeri , 

Great job reviewing a project from the past! 🍾 🎆 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Great job identifying things that could be improved. 💯 
✅ Great job providing a detailed review summary!📜 

## Required Changes ♻️

- [ ] I noticed that you suggested some changes to the semantics of the project. ♻️ Please, consider using GitHub's inline comments inside their review features to point out exactly where you're requiring this change. It is important to be as clear as you can with the changes you request. This will help enormously the code's author to find where he has to make this change and why. 🤓  You could refer to this [tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) from the main docs. 🛠️  

_**NOTE:** Be careful with the 7th step. Instead of clicking on "Add a single comment", since you're doing a review, you should click "Start review"_

- [ ] Also, please consider giving an explanation for each change requested. 

>From your review: `It's better to have a descriptive README.md file.`

Why is adding a descriptive README file better? ❓  Including this explanation will help the code's author to improve their knowledge and to include those changes in future projects. Please, add this! 🛠️ 


_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]:** It would be a great idea to include emojis and gifs 🤖 in your review comments! They make the code review more human-friendly 👽 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
     <p id="description">
       Thank you for taking the time to help us improve the platform
```
Comment (billymohajeri) [inline index.html:12]: 123

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "codingwithsalwa3",
+  "version": "1.0.0",
+  "description": "**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.",
+  "main": "index.js",
```
Comment (billymohajeri) [inline package.json:5]:
```suggestion
  "main": "index.js",
```

File: index.html [hunk 3]
```diff
@@ -11,7 +11,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
...
     </p>
-  
+   
     <form id="survey-form">
```
Comment (billymohajeri) [inline index.html:15]:
```suggestion
    <section>
    <form id="survey-form">
```

**Review by billymohajeri — COMMENTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ♻️ 
Hi @billymohajeri, 

Good job so far! 👍🎇



You still need to work on some issues to go to the next project, but you are almost there! 😎🤩 Just a few steps! 👣🤏

## Highlights 💯 
✅ Informative commit messages
✅ GitHub flow used
✅ Good PR

## Required Changes ♻️
- You have some errors ❌ in stylelint. Please solve those.

- It's better to have a descriptive README.md file.
  So the main idea is to have documentation 📄 for your project 💻 and help others who don't know anything about your project, have a general idea about it. 💡
  A Microverse README.md template is available [here 🔗](https://github.com/microverseinc/readme-template) that you can customize.

- It's better to use semantic HTML tags and put every different block of codes in specified sections. Writing semantic HTML makes your code easier to understand, making the source code more readable for other developers. 🤓 You can also take a look at [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

👉_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (billymohajeri) [inline index.html:15]:
```suggestion
    <section class="survey-form">
    <form id="survey-form">
```

For example, you can use a _section_ tag and put _form_ tag in it. Or you can add a _header_ tag and put your _h1_ and _p_ tags in it.
These changes increase your code's readability. So it clearly communicates its intent to the reader. Most likely, the code we write will be read by other developers, who will either want to understand or modify the way our code works.

**Review by billymohajeri — COMMENTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ♻️ 
Hi @billymohajeri, 

Good job so far! 👍🎇



You still need to work on some issues to go to the next project, but you are almost there! 😎🤩 Just a few steps! 👣🤏

## Highlights 💯 
✅ Informative commit messages
✅ GitHub flow used
✅ Good PR

## Required Changes ♻️
- You have some errors ❌ in stylelint. Please solve those.

- It's better to have a descriptive README.md file.
  So the main idea is to have documentation 📄 for your project 💻 and help others who don't know anything about your project, have a general idea about it. 💡
  A Microverse README.md template is available [here 🔗](https://github.com/microverseinc/readme-template) that you can customize.

- It's better to use semantic HTML tags and put every different block of codes in specified sections. Writing semantic HTML makes your code easier to understand, making the source code more readable for other developers. 🤓 You can also take a look at [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

👉_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (billymohajeri) [inline index.html:15]:
```suggestion
    <section class="survey-form">
    <form id="survey-form">
```

For example, you can use a _section_ tag and put _form_ tag in it. Or you can add a _header_ tag and put your _h1_ and _p_ tags in it.

These changes increase your code's readability. So it clearly communicates its intent to the reader. Most likely, the code we write will be read by other developers, who will either want to understand or modify the way our code works.

**Review by billymohajeri — COMMENTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ♻️ 
Hi @billymohajeri, 

Good job so far! 👍🎇



You still need to work on some issues to go to the next project, but you are almost there! 😎🤩 Just a few steps! 👣

## Highlights 💯 
✅ Informative commit messages
✅ GitHub flow used
✅ Good PR

## Required Changes ♻️
- You have some errors ❌ in stylelint. Please solve those.

- It's better to have a descriptive README.md file.
  So the main idea is to have documentation 📄 for your project 💻 and help others who don't know anything about your project, have a general idea about it. 💡
  A Microverse README.md template is available [here 🔗](https://github.com/microverseinc/readme-template) that you can customize.

- It's better to use semantic HTML tags and put every different block of codes in specified sections. Writing semantic HTML makes your code easier to understand, making the source code more readable for other developers. 🤓 You can also take a look at [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

👉_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @billymohajeri  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## Highlights
📌 Good to use inline comments for your reviews :heavy_check_mark: 
📌 Good to use a gif in your review summary :heavy_check_mark: 
📌 Good communication :heavy_check_mark: 

## Optional suggestions

- [ ] I think it would be better if you use emojis in your inline comments to make your review human-friendly.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**